MELA/fortran/mod_Parameters.F90

```diff
@@ -464,6 +467,190 @@ MODULE ModParameters
    real(8), public :: M_Vprime
    real(8), public :: Ga_Vprime
 
+   
+   
+   
+   
+!-- second resonance (H2) couplings for off-shell VBF
+!-- HVV' couplings to ZZ/ZA/AA and WW
+   complex(8), public :: gh2z1 = (0.0d0,0d0) 
+   complex(8), public :: gh2z2 = (0.0d0,0d0)
+   complex(8), public :: gh2z3 = (0.0d0,0d0)
+   complex(8), public :: gh2z4 = (0.0d0,0d0)   ! pseudoscalar
+
+   complex(8), public :: gh2zgs2  = (0.00d0,0d0)
+   complex(8), public :: gh2zgs3  = (0.00d0,0d0)
+   complex(8), public :: gh2zgs4  = (0.00d0,0d0)
+   complex(8), public :: gh2gsgs2 = (0.00d0,0d0)
+   complex(8), public :: gh2gsgs3 = (0.00d0,0d0)
+   complex(8), public :: gh2gsgs4 = (0.00d0,0d0)
+
+!-- HVV contact terms
+!    logical, public :: UseVprime=.false.  ! false: contact interaction,  true: heavy Zprime propagator
+!    integer, public :: OnlyVVpr = 0       ! +1=only contact term for decay1, -1=only contact term for decay2, 0=all terms
+   complex(8), public :: eh2z_L_E  = (0.00d0,0d0)
```

Comment:
The second resonance contact terms should be removed. It is unclear how they will be implemented in MCFM, and their names are not consistent with first resonance.